src/_scss/pages/aboutTheData/aboutTheData.scss

```diff
@@ -56,4 +56,25 @@
             margin-right: rem(5);
         }
     }
+    .usda-table {
+        thead {
+            th {
+                .wide {
+                    .tooltip-spacer {
+                        width: 600px !important;
+                    }
+                    &.wide_left {
+                        .tooltip-spacer {
+                            left: -408px !important;
+                        }
+                    }
+                    &.wide-right {
+                        .tooltip-spacer {
+                            right: 408px !important;
```

Comment:
```suggestion
                    &.wide_left {
                        .tooltip-spacer {
                            height: rem(450);
                            overflow-y: scroll;
                        }
                    }
```

Reply:
This line is adjusting the position `right` rather than height. I'm using important to override inline style coming from the component library.

Reply:
Were you suggesting scrolling w/in the tooltips instead? I can check with design (I'm personally not a fan of that UX)